package.json

```
@@ -53,7 +70,8 @@
     "url": "https://github.com/implydata/druid-query-toolkit"
   },
   "files": [
-    "build/"
+    "dist/",
+    "types/"
```

Optional suggestion, but you might consider including `src` along with the output files. Since the `declarationMap` option is enabled in tsconfig, when someone does a Cmd+B/F12/"navigate to source" action in their IDE, it will take them all the way to the TypeScript source instead of the definition files or the JS output.

The cost is a larger npm tarball, but it's a nice DX enhancement for those who want to know how the library is implemented without cloning the repo or going to GitHub.

```suggestion
    "dist/",
    "src/",
    "types/"
```

---

great idea.